docs/sources/alerting/fundamentals/_index.md

```diff
@@ -105,11 +110,15 @@ A rule using the PromQL expression above creates as many alert instances as the
 
 [Contact points](ref:contact-points) determine the notification message and where notifications are sent. For example, you might have a contact point that sends notifications to an email address, to Slack, to an incident management system (IRM) such as Grafana OnCall or Pagerduty, or to a webhook.
 
+### Notification messages
+
+Notification messages can be customized. The default notification messages include common alert details, which are usually sufficient. Set alert annotations to help responders address alert issues.
```

Comment:
```suggestion
Notification messages can be customized. The default notification messages include common alert details, which are usually sufficient. Set alert annotations to help responders address alert issues.
```

Reply:
what do you mean by "which are usually sufficient"? - I would take that out and switch the sentence around. Default notification messages include common alert details, such as xxxx. They can also be customized if needed.

Reply:
+1, should be fixed by https://github.com/grafana/grafana/pull/95987/commits/3748163f659d55bbed1411945615af973794e1e0